src/components/Input/Input.scss

```
@@ -0,0 +1,13 @@
+input {
+  display: flex;
+  height: 40px;
+  width: 40px;

```

```suggestion
  width: 100%;
```
Let's keep the width decision to the place that we'll put it in.